### PR TITLE
HEP: Enhanced Maintenance Mode

### PR DESCRIPTION
**Pull request by votdev**
Related to: #5069

Signed-off-by: Volker Theile <vtheile@suse.com>

### CODE REVIEW

File: enhancements/20240605-enhanced-maintenance-mode.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+time="2024-03-19T13:01:19Z" level=info msg="restarting the VM that was temporary shut down for maintenance mode" namespace=default virtualmachine_name=test01
+```
+
+#### Case 3
+1. Add the `harvesterhci.io/maintain-mode-strategy` label to a VM of your choice. Press `Edit YAML` and add `harvesterhci.io/maintain-mode-strategy: Shutdown` to `metadata.labels`. Save the settings.
+2. Make sure a second VM without the `harvesterhci.io/maintain-mode-strategy` label is running on the same node. Migrate it to the node on which the other VM is running if necessary.
+3. Note the node on which the VMs are running.
+4. Go to the `Hosts` page and select the `Enable Maintenance Mode` action in the actions dropdown menu of the node your VMs are running on.
+5. After some time the `Cordon` state should be displayed for the host we are putting into maintenance mode.
+6. Now go to the `Virtual Machines` page. The VM with the `harvesterhci.io/maintain-mode-strategy` label should show the `Stopping` label. The other VM should show the `Migrate` status badge.
```
Comment (w13915984028) [inline enhancements/20240605-enhanced-maintenance-mode.md]: show the `Stopping` label  
or
show the `Stopping` status badge?

**Review by w13915984028 — APPROVED** [review]
Nice enhancement with very detailed user stories, thanks.

File: enhancements/20240605-enhanced-maintenance-mode.md (hunk 1 above)
Comment (votdev) [reply enhancements/20240605-enhanced-maintenance-mode.md]: I changed `label` => `status badge` because this should be much clearer to the user.

**Review by ibrokethecloud — APPROVED** [review]
lgtm. thanks.

**Review by bk201 — APPROVED** [review]

**Review by FrankYang0529 — APPROVED** [review]

**Pull request merged**